ignite-generator/src/app/index.es

```diff
@@ -251,61 +268,21 @@ export class AppGenerator extends Generators.Base {
     const useCustomRepo = typeof requestedRepo !== 'undefined' && requestedRepo !== null && requestedRepo !== ''
     // the right repo to use
     const repo = useCustomRepo ? requestedRepo : defaultRepo
+    const branch = this._getGitBranch()
     // start spinner and message
-    const animation = Utilities.startStep(`Downloading latest Ignite files from ${repo}`, this)
+    const animation = Utilities.startStep(`Downloading latest Ignite files from ${repo}#${branch}`, this)
 
     const done = this.async()
     const command = 'git'
-    const commandOpts = ['clone', repo, this.sourceRoot()]
+    const commandOpts = ['clone', '--depth', '1', '--branch', branch, repo, this.sourceRoot()]
```

Comment:
This is awesome, so this will take care of tags and branches from what I'm reading!